app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -426,9 +429,11 @@ private void showMenu() {
     }
 
     private void updateMenu() {
+        boolean isMyShotUnreadEnabled = AppConfigWrapper.getMyshotUnreadEnabled(this);
+        boolean showUnread = isMyShotUnreadEnabled && Settings.getInstance(this).hasUnreadMyShot();
```

Comment:
nit: final?

Reply:
I'm king of not adding final 😈  Thanks!!